tests/phpunit/tests/block-template-utils.php

```diff
@@ -377,4 +377,160 @@ function test_wp_generate_block_templates_export_file() {
 		}
 		$this->assertTrue( $has_html_files, 'contains at least one html file' );
 	}
+
+	/**
+	 * Helper function to get the Template Hierarchy for a given slug.
+	 *
+	 * @ticket 56467
+	 */
+	public function test_get_template_hierarchy() {
```

Comment:
For each assertion in this test:

* Use `assertSame()` instead of `assertEquals`
* Add an assertion message (3rd arg) to describe it. Why? If an assertion fails, the rest will not run. By adding an assertion message, it helps to identify which assertion failed and why.

Reply:
I updated all the test suggestions. Thanks for explaining the `why` for multiple assertions.